module/zfs/zfs_ioctl.c

```diff
@@ -1683,7 +1684,13 @@ zfs_ioc_pool_scan(zfs_cmd_t *zc)
 	if ((error = spa_open(zc->zc_name, &spa, FTAG)) != 0)
 		return (error);
 
-	if (zc->zc_cookie == POOL_SCAN_NONE)
+	if (zc->zc_flags < POOL_SCRUB_NORMAL ||
```

Comment:
This can never be true since `zc_flags` is unsigned and `POOL_SCRUB_NORMAL = 0`.  Let's drop this first check, it's going to upset static code analysis utilities.